apps/web/src/features/layout/components/Starfield/starfield.module.css

```
@@ -892,24 +1042,40 @@
   gap: 8px;
   box-shadow: 0 8px 32px rgba(157, 78, 221, 0.3);
   transition: all 0.3s ease;
-  font-family: 'Inter', -apple-system, BlinkMacSystemFont, sans-serif;
+  font-family:
+    "Inter",
```

Stylelint (17.14.0)

[error] Expected no quotes around "Inter" (font-family-name-quotes)

(font-family-name-quotes)